bucket/siyuan-note.json

```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "https://github.com/siyuan-note/siyuan",
+    "description": "SiYuan is a Markdown Block-Reference and Bidirectional-Link note-taking application",
+    "version": "1.4.8",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://b3log.org/siyuan/eula.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/siyuan-note/siyuan/releases/download/v1.4.8/siyuan-1.4.8-win.exe#/siyuan.7z",
+            "hash": "2e72af2c6211595ce4e47c2141e029ce53fb7f934d383f9d8f79493ac6a70b61",
+            "pre_install": [

```

Comment:
Does it not work without the pre-install block?

Reply:
Thanks for the good check!
It not work without the pre-install block